apps/codeforafrica/src/config/index.js

```diff
@@ -0,0 +1,52 @@
+import facebook from "@/codeforafrica/assets/images/facebook.png";
```

Comment:
Instead of using `config`, use `lib/index.js` to return expected props. We then only have one place to change when we switch to the back-end CMS. Otherwise, `config` always end up with unused imported assets, etc.